test/functional/helpers.lua

```diff
@@ -49,6 +53,10 @@ end
 
 local session, loop_running, loop_stopped, last_error
 
+local function set_session(s)
+  session = s
+end
```

Comment:
@bfredl @tarruda This is a temporary hack to allow multiple spawned sessions in a single test. After this PR I plan to refactor helpers.lua into a module with a `new()` constructore so that a test can control multiple sessions. Does that sound ok?

Another thing that would be useful (for testing swapfiles) is ability to `kill -9` the session. Any nice way to do that? I see that `loop.c` in the lua client [does not send signals](https://github.com/neovim/lua-client/blob/master/nvim/loop.c#L58).